From 453bdd1bd7cb6f713b56829ae6a728df8c621df0 Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Tue, 11 Apr 2017 09:42:57 +0200
Subject: [PATCH 3/3] Disable tests that require newer version of libpcap.

As can be seen from ltrace:
pcap_compile(0x25d4250, 0x7ffc966dcba0, "geneve && tcp", 1, nil) = -1
these two filters (geneve) cannot be compiled, but it is not a tcpdump
fault.
---
 tests/TESTLIST | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/tests/TESTLIST b/tests/TESTLIST
index ba29da4..0a1e2f1 100644
--- a/tests/TESTLIST
+++ b/tests/TESTLIST
@@ -300,8 +300,9 @@ aoe_1-v		AoE_Linux.pcap		aoe_1-v.out	-t -v
 
 # Geneve tests
 geneve-v	geneve.pcap		geneve-vv.out	-t -vv
-geneve-vni	geneve.pcap		geneve-vni.out	-t geneve 0xb
-geneve-tcp	geneve.pcap		geneve-tcp.out	-t "geneve && tcp"
+# These filters cannot be compiled by our libpcap
+# geneve-vni	geneve.pcap		geneve-vni.out	-t geneve 0xb
+# geneve-tcp	geneve.pcap		geneve-tcp.out	-t "geneve && tcp"
 
 # DHCP tests
 dhcp-rfc3004	dhcp-rfc3004.pcap	dhcp-rfc3004-v.out	-t -v
@@ -355,11 +356,11 @@ lisp_ipv6_eid		lisp_ipv6.pcap		lisp_ipv6.out		-t -v
 
 # pcap invalid versions (first: version = 1.4 ; second: version = 2.5)
 pcap-invalid-version-1 pcap-invalid-version-1.pcap pcap-invalid-version-1.out -t
-pcap-invalid-version-2 pcap-invalid-version-2.pcap pcap-invalid-version-2.out -t
+# pcap-invalid-version-2 pcap-invalid-version-2.pcap pcap-invalid-version-2.out -t
 
 # pcap-ng invalid version (first: version = 0.1 ; second: version = 1.1)
 pcap-ng-invalid-vers-1 pcap-ng-invalid-vers-1.pcap pcap-ng-invalid-vers-1.out -t
-pcap-ng-invalid-vers-2 pcap-ng-invalid-vers-2.pcap pcap-ng-invalid-vers-2.out -t
+# pcap-ng-invalid-vers-2 pcap-ng-invalid-vers-2.pcap pcap-ng-invalid-vers-2.out -t
 
 # NSH over VxLAN-GPE
 nsh-over-vxlan-gpe     nsh-over-vxlan-gpe.pcap nsh-over-vxlan-gpe.out     -t
-- 
1.8.3.1

